pkg/generator/generator_test.go

```diff
@@ -6,7 +6,7 @@ import (
 	"testing"
 
 	"github.com/operator-framework/combo/pkg/combination"
-	"github.com/operator-framework/combo/test/assets/generatorTestData"
+	testdata "github.com/operator-framework/combo/test/assets/generator"
```

Comment:
In go, the canonical directory and package name to use for test data is `testdata`.

Reply:
+1 here as well, despite this being out of scope for this PR (imo), I'd still like to see this get addressed sometime in the near future.